File: android/src/main/java/com/example/appwheel_flutter/AWResponse.java
package com.example.appwheel_flutter;

import java.io.Serializable;

/**
 * @author: Mr.H
 * @date: 2021/11/11
 */
public class AWResponse <T> implements Serializable {

    public boolean result;
    public T data;
    public String msg;

    public AWResponse(boolean result, T data, String msg) {
        this.result = result;
        this.data = data;
        this.msg = msg;
    }
}

File: android/src/main/kotlin/com/example/aw_purchase/AWResponse.kt
package com.example.aw_purchase

/**

@author: Mr.H
@date: 2021/10/28
 */
data class AWResponse <T>(val result: Boolean,
                      val data: T,
                      var msg: String) {
//    override fun toString(): String {
//        msg = if (msg.isNullOrEmpty()){""}else{msg}
//        return "{\"result\":$result,\"data\":${if(data.isNotEmpty()){data}else{"\"\""}}," +
//                "\"msg\":$msg}"
//    }
}
